standup/README.md

```diff
@@ -0,0 +1,44 @@
+# Firestore Standup collection data model
+
+```json
+{
+  "id": "String",          // Unique identifier for the document
+  "type": "String",        // Type of the document, e.g., "task progress" or "user progress"
+  "userId": "String",      // User ID (applicable for both task progress and user progress)
+  "taskId": "String",      // Task ID (applicable for task only)
+  "completed": "String",   // Completed portion of the task (applicable for both similar to yesterday or past progress for user)
+  "planned": "String",     // Planned portion of the task (applicable for both similar to today or upcoming plan for user)
+  "blockers": "String",    // Blockers (applicable for both task and user progress)
+  "date": "Timestamp",     // Date of the standup or user progress (applicable for user progress)
+  "createdAt": "Timestamp" // Creation timestamp (applicable for both)
+}
+
+```
+
+## Example
+User Progress Update
+```json
+{
+  "id": "d4606ee112f7892c45a0",
+  "type": "task",
+  "userId": "f042d216571a40ece689",
+  "taskId": "a52c1d09780ee641f462",
```

Comment:
is there any need to provide the taskId if we already have the id key present?

Reply:
the `id` here is the document id of the progress document.

Reply:
so from where are we getting this taskId then?

Reply:
it will be the document id that is autogenerated.